examples/evaluate/03-scalarDROM.py

```diff
@@ -1,16 +1,16 @@
 """.. _ref_example_scalarDROM:
 
-Scalar dynamic ROM Twin evaluation example
-------------------------------------------
-This example shows how you can use PyTwin to load and evaluate a Twin model.
-The model is a scalar dynamic ROM created out of a 3D thermal model of a
-Heat Exchanger, having a heat flow as input and three temperature probes
-as outputs. The example shows a workflow for what-if analysis by deploying
-a second twin in parallel while simulating the original twin and comparing
-the different predictions. This is done using the specific functions for saving
-and loading the twin states. It also illustrates the usage of modify_pytwin_working_dir
-to change the default working directory location (%temp%) to a user specified location
-where the different logging files will be available.
+Twin evaluation of a scalar dynamic ROM
+---------------------------------------
+This example shows how you can use PyTwin to load and evaluate a twin model
+of a scalar dynamic ROM. The ROM is created from a 3D thermal model of a
+heat exchanger. The input is the heat flow. The outputs are the temperatures
+from three temperature probes. The workflow for this example performs what-if
+analysis by deploying a second twin in parallel while simulating the original twin
+so that results from the different predictions can be compared. This comparison
+is done using functions for saving and loading twin states. This example also
```

Comment:
```suggestion
is done using methods for saving and loading twin states. This example also
```
I was told that in Python terminology, methods are functions that belong to a class. Functions that don't belong to a class are not methods. I used methods since I believe this refers to functions that belong to a class.

Reply:
@PipKat Yes, you should always refer to the [python doc glossary](https://docs.python.org/3/glossary.html) for this type of questions. You can compare [function](https://docs.python.org/3/glossary.html#term-function) and [method](https://docs.python.org/3/glossary.html#term-method) definitions. So here, the terminology function was the most relevant. I will use it in all examples.